node/internal/actors/mock_node_test.go

```diff
@@ -0,0 +1,87 @@
+package actors
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/nats-io/nkeys"
+	"github.com/synadia-io/nex/models"
+	actorproto "github.com/synadia-io/nex/node/internal/actors/pb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+const (
+	NodeXkey    string = "SXAAJQZR4DLEWWUAIDYHIV3JRXNHYMKV5O3POAWXQLATMPAF2OWLJTZ4BM"
+	NodeXkeyPub string = "XC6LBRSGPEQTOFBYEII2DX2RVWXEK27BARNJMX2V3WIKR6XIWFLRYYZY"
+)
+
+type mockNode struct {
+	options *models.NodeOptions
+}
+
+func NewMockNode() *mockNode {
+	return &mockNode{
+		options: &models.NodeOptions{
+			Tags: map[string]string{models.TagNodeName: "mock-node", models.TagNexus: "TESTNEXUS"},
+		},
+	}
+}
+
+func (n *mockNode) Auction(auctionId string, agentType []string, tags map[string]string) (*actorproto.AuctionResponse, error) {
+	return &actorproto.AuctionResponse{
+		BidderId:   "abc123",
+		Version:    "0.0.0",
+		TargetXkey: NodeXkeyPub,
+		StartedAt:  timestamppb.New(time.Now()),
+		Tags:       n.options.Tags,
+		Status:     make(map[string]int32),
+	}, nil
+}
+
+func (n mockNode) Ping() (*actorproto.PingNodeResponse, error) {
+	return &actorproto.PingNodeResponse{
+		NodeId:        "NNODE123",
+		Version:       "0.0.0",
+		TargetXkey:    "XNODEXKEY",
+		StartedAt:     timestamppb.Now(),
+		Tags:          n.options.Tags,
+		RunningAgents: map[string]int32{},
```

Comment:
Nit, this is using the literal syntax where as above uses `make(..)`. Should be consistent throughout.